### DISCUSSION TITLE
Welcome to blendercam Discussions!

### DISCUSSION THREAD

**Discussion by pppalain** [pppalain/blendercam#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cheapsk8studios** [comment]
When trying to calculate 5 axis toolpath, it does not work and gives an error. Here is a copy from the log:

bpy.ops.object.calculate_cam_path()
bpy_prop_collection: attribute "active" not found


Fabexcnc 1.0.66
Blender 4.2.8 (portable)
Winows 10 Pro x64

Glenn
Cheapsk8 Studios LLC
